handler.go

```diff
@@ -0,0 +1,109 @@
+package solidproxy
+
+import (
+	"crypto/tls"
+	"log"
+	"net/http"
+	"sync"
+
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
+)
+
+var (
+	Logger             *log.Logger
+	agentWebID         string
+	userWebID          string
+	insecureSkipVerify bool
+
+	cookies  = map[string]map[string][]*http.Cookie{}
+	cookiesL = new(sync.RWMutex)
+)
+
+// NewServer creates a new server handler
+func NewProxyHandler(config *ServerConfig) *echo.Echo {
+	Logger.Println("\n---- starting proxy server ----")
+	Logger.Printf("config: %#v\n", config)
+
+	// set local variables used by the proxy client
+	agentWebID = config.Agent
```

Comment:
I'm thinking if I still want to force the User in case it is provided in the config params. It will most likely be gone in the next patch.